crates/egui/src/memory.rs

```diff
@@ -168,24 +168,28 @@ impl FocusDirection {
 #[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
 #[cfg_attr(feature = "serde", serde(default))]
 pub struct Options {
-    /// The default style for new [`Ui`](crate::Ui):s.
+    /// The default style for new [`Ui`](crate::Ui):s in dark mode.
     #[cfg_attr(feature = "serde", serde(skip))]
-    pub(crate) style: std::sync::Arc<Style>,
+    pub(crate) dark_style: std::sync::Arc<Style>,
 
-    /// Whether to update the visuals according to the system theme or not.
+    /// The default style for new [`Ui`](crate::Ui):s in light mode.
+    pub(crate) light_style: std::sync::Arc<Style>,
+
+    /// A preference for how to select between [`crate::Context::dark_style()`] and [`crate::Context::light_style()`]
+    /// as the active style used by all subsequent windows, panels etc.
     ///
-    /// Default: `true`.
-    pub follow_system_theme: bool,
+    /// Default: `ThemePreference::System`.
+    pub theme_preference: ThemePreference,
 
-    /// Which theme to use in case [`Self::follow_system_theme`] is set
+    /// Which theme to use in case [`Self::theme_preference`] is [`ThemePreference::System`]
     /// and egui fails to detect the system theme.
     ///
     /// Default: [`crate::Theme::Dark`].
     pub fallback_theme: Theme,
 
     /// Used to detect changes in system theme
```

Comment:
Yes, the deciding thing for the theme is ThemePreference which has both a `Dark` and a `Light` variant for forcing a theme.

If ThemePreference is set to `System`, then `fallback_theme` is the fallback in case we don't have a system theme [source](https://github.com/bash/egui/blob/2ea927dbac6a3102381e86d17e0b54f23e2905f6/crates/egui/src/memory/mod.rs#L313-L319).